Src/Web/Web.Shared.Net/RequestTrackingTelemetryModule.cs

```diff
@@ -267,6 +267,13 @@ public void Initialize(TelemetryConfiguration configuration)
             this.telemetryClient = new TelemetryClient(configuration);
             this.telemetryClient.Context.GetInternalContext().SdkVersion = SdkVersionUtils.GetSdkVersion("web:");
 
+            this.DisableTrackingProperties = configuration.EvaluateExperimentalFeature("deferRequestTrackingProperties");
+            if (this.DisableTrackingProperties)
+            {
+                configuration.TelemetryProcessorChainBuilder.Use(next => new PostSamplingTelemetryProcessor(next));
```

Comment:
can you validate that this achieves the intended effect? I believe this should be done on default sink's TP instead on common TP.

Reply:
Done. wrote a bunch of new unit tests. :)